eden/scm/tests/run-tests.py

```diff
@@ -697,7 +697,7 @@ def parseargs(args, parser):
     if options.local:
         testdir = os.path.dirname(canonpath(sys.argv[0]))
         reporootdir = os.path.dirname(testdir)
-        pathandattrs = [("hg", "with_hg")]
+        pathandattrs = [("sl", "with_hg")]
```

Comment:
We haven't migrate all internal usages to `sl` yet, this would break internal `./run-tests.py`

One way to fix this is to make line 701 ~709 to try both `[("sl", "with_hg"), ("hg", "with_hg")]`, and choose the available one; if both of them doesn't exist, then raise error.

Reply:
This part of the patch was part of PR #458. I pushed a fix to the issue you mentioned in that PR.